modules/luci-mod-network/htdocs/luci-static/resources/view/network/dhcp.js

```diff
@@ -614,16 +671,34 @@ return view.extend({
 			so.value(ipv4, ipaddrs[ipv4] ? '%s (%s)'.format(ipv4, ipaddrs[ipv4]) : ipv4);
 		});
 
-		so = ss.option(form.Value, 'leasetime', _('Lease time'));
+		so = ss.option(form.Value, 'leasetime', 
+			_('Lease time'),
+			_('Host-specific lease time, e.g. <code>2m</code>, <code>3h</code>, <code>5d</code>.'));
 		so.rmempty = true;
 
-		so = ss.option(form.Value, 'duid', _('DUID'));
+		so = ss.option(form.Value, 'duid',
+			_('DUID'),
+			_('The DHCPv6-DUID of this host.'));
 		so.datatype = 'and(rangelength(20,36),hexstring)';
 		Object.keys(duids).forEach(function(duid) {
 			so.value(duid, '%s (%s)'.format(duid, duids[duid].hostname || duids[duid].macaddr || duids[duid].ip6addr || '?'));
 		});
 
-		so = ss.option(form.Value, 'hostid', _('IPv6 suffix (hex)'));
+		so = ss.option(form.Value, 'hostid',
+			_('IPv6-Suffix (hex)'),
+			_('The IPv6 interface identifier (address suffix) as hexadecimal number (max. 8 chars).'));
+
+		so = ss.option(form.Value, 'tag',
+			_('Tag'),
+			_('Set the given tag for matching hosts.'));
+
+		so = ss.option(form.Flag, 'broadccast',
+			_('Tag'),
```

Comment:
Both options use the `Tag` caption, is this intentional? Shouldn't the latter use something like `_('Broadcast')` as caption? Also the `broadccast` option name looks like a type.

Reply:
Yeah, held hostage by my keyboard..... needs fixing 🤦 

Fixed.